scripts/gtk-bundle-from-msys2.sh

```diff
@@ -135,7 +137,8 @@ _getpkg() {
 	if [ "$use_cache" = "yes" ]; then
 		package_info=$(pacman -Qi mingw-w64-$ABI-$1)
 		package_version=$(echo "$package_info" | grep "^Version " | cut -d':' -f 2 | tr -d '[[:space:]]')
-		ls $cachedir/mingw-w64-${ABI}-${1}-${package_version}-* | sort -V | tail -n 1
+		# use @(gz|xz|zst) to filter out signature files (e.g. mingw-w64-x86_64-...-any.pkg.tar.zst.sig)
+		ls $cachedir/mingw-w64-${ABI}-${1}-${package_version}-*.tar.@(gz|xz|zst) | sort -V | tail -n 1
```

Comment:
With `*.tar.{gz,xz,zst}` it should be possible to do without extglob shell option.

Reply:
For me, it doesn't work:
```bash
enrico@endor:/tmp/shglob$ ls
test-1.tar.zst
enrico@endor:/tmp/shglob$ ls test-*.tar.{gz,xz,zst}
ls: cannot access 'test-*.tar.gz': No such file or directory
ls: cannot access 'test-*.tar.xz': No such file or directory
 test-1.tar.zst
enrico@endor:/tmp/shglob$ shopt -s extglob
enrico@endor:/tmp/shglob$ ls test-*.tar.@(gz|xz|zst)
test-1.tar.zst
```

`*.tar.{gz,xz,zst}` tries to expand to all of the patterns and so fails for missing files. The `extglob` variant only matches for *existing* files.
I don't see a big issue here, we already require `bash` anyway and performance is also not that relevant as probably nearly all other operations in the script are way more expensive.

Reply:
I'm not familiar with that syntax, but sure it's probably as readable as the grep variant, and it's fine regarding your remarks